integration-tests/testkit/seed.ts

```diff
@@ -57,21 +50,29 @@ import {
   updateSchemaVersionStatus,
   updateTargetValidationSettings,
 } from './flow';
+import {
+  OrganizationAccessScope,
+  ProjectAccessScope,
+  ProjectType,
+  RegistryModel,
+  SchemaPolicyInput,
+  TargetAccessScope,
+} from './gql/graphql';
 import { execute } from './graphql';
 import { UpdateSchemaPolicyForOrganization, UpdateSchemaPolicyForProject } from './schema-policy';
 import { collect, CollectedOperation, legacyCollect } from './usage';
 import { generateUnique } from './utils';
 
 export function initSeed() {
-  const pg = {
-    user: ensureEnv('POSTGRES_USER'),
-    password: ensureEnv('POSTGRES_PASSWORD'),
-    host: ensureEnv('POSTGRES_HOST'),
-    port: ensureEnv('POSTGRES_PORT'),
-    db: ensureEnv('POSTGRES_DB'),
-  };
-
   function createConnectionPool() {
+    const pg = {
+      user: ensureEnv('POSTGRES_USER'),
+      password: ensureEnv('POSTGRES_PASSWORD'),
+      host: ensureEnv('POSTGRES_HOST'),
+      port: ensureEnv('POSTGRES_PORT'),
+      db: ensureEnv('POSTGRES_DB'),
+    };
```

Comment:
Delaying this assertion because it is not required for seeding within the e2e tests.